caluma/workflow/tests/snapshots/snap_test_task.py

```diff
@@ -47,6 +47,66 @@
     }
 }
 
+snapshots["test_query_all_tasks[multiple_instance_complete_task_form] 1"] = {
+    "allTasks": {
+        "edges": [
+            {
+                "node": {
+                    "__typename": "MultipleInstanceCompleteTaskFormTask",
```

Comment:
I assume this is a deprecated snapshot? Good to run all tests with --snapshot-update to remove deprecations.

Reply:
Indeed, fixed.